tests_pytest/_fw_tests_common_base/base_ru_integration_test.py

```diff
@@ -85,6 +88,44 @@ def build_tpc():
     return tpc, linear_w_min_nbit, linear_a_min_nbit, default_w_nbit, default_a_nbit, binary_out_a_bit
 
 
+def build_snc_tpc():
```

Comment:
Need to consider how to test this feature, through a fusing pattern or using a default TPC, because current TPCs don't include this fusing.

Reply:
The next PR contains e2e tests that will include this scenario.